src/main/java/org/apache/aurora/scheduler/filter/SchedulingFilterImpl.java

```diff
@@ -95,6 +95,11 @@ public int compare(IConstraint a, IConstraint b) {
     for (IConstraint constraint : VALUES_FIRST.sortedCopy(taskConstraints)) {
       Optional<Veto> veto = ConstraintMatcher.getVeto(jobState, offerAttributes, constraint);
       if (veto.isPresent()) {
+        // If we have specified { dedicated: */* } then it is a green light to run anywhere
+        // Only check if veto'd which shouldn't hurt scheduling time too much
+        if (constraint.getName().equals(DEDICATED_ATTRIBUTE) && constraint.getConstraint().getValue().getValues().contains("*/*")) {
```

Comment:
would this be more clear if we move this logic into the constraint matcher to make it not create a veto if the value is */* ?